packages/back-end/src/controllers/organizations.ts

```diff
@@ -865,3 +865,36 @@ export async function putUpload(req: Request, res: Response) {
     status: 200,
   });
 }
+
+export async function putAdminResetUserPassword(
+  req: AuthRequest<{
+    userToUpdateId: string;
+    updatedPassword: string;
+  }>,
+  res: Response
+) {
+  const { updatedPassword } = req.body;
+  const userToUpdateId = req.params.id;
+
+  if (IS_CLOUD) {
```

Comment:
Not sure if this check is completely necessary or if there is some abstracted logic already handling this.